firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -61,6 +62,7 @@
 
   private static final String X_ANDROID_PACKAGE_HEADER_KEY = "X-Android-Package";
   private static final String X_ANDROID_CERT_HEADER_KEY = "X-Android-Cert";
```

Comment:
just because I see this here:
that's great.
have you already implemented sending the App restrictions header?

Reply:
Yes, I have